lib/wasm/WebAssemblyGenerator.js

```diff
@@ -225,7 +227,7 @@ const rewriteImportedGlobals = state => bin => {
 
 				node.init = [
 					// Poisong globals, they are meant to be rewritten
-					t.objectInstruction("const", valtype, [t.numberLiteralFromRaw(666)])
+					t.objectInstruction("const", valtype, [t.numberLiteralFromRaw(0)])
```

Comment:
That was useful for testing, do you think we shouldn't do it? (also comment above)

Reply:
It increases the size of the wasm. But we can use `66`.